routes/web.php

```diff
@@ -21,8 +21,8 @@
 
 $router->get("/", HomeController::class)->name("home");
 $router->get($uri->translate("about"), AboutController::class)->name("about");
-$router->get($uri->translate("case-studies"), [CaseStudiesController::class, "index"])->name("case-studies");
-$router->get($uri->translate("case-studies/{slug}"), [CaseStudiesController::class, "get"])->name("case-studies.entry");
+$router->get($uri->translate("projects"), [CaseStudiesController::class, "index"])->name("projects");
```

Comment:
I would like to unify naming: projects or case studies.

Reply:
Done